daemon/graphdriver/fsdiff.go

```diff
@@ -35,6 +35,7 @@ type NaiveDiffDriver struct {
 //     Changes(id, parent string) ([]archive.Change, error)
 //     ApplyDiff(id, parent string, diff archive.Reader) (size int64, err error)
 //     DiffSize(id, parent string) (size int64, err error)
+// Note that these assume a local file system for the graph driver (does not support LCOW)
```

Comment:
I would just mention in the description that the `NaiveDiffDriver` operates by calculating and applying diffs using the local file system. Using the naive diff is a choice of the graph driver implementer, no need to mention LCOW.